app/submitter/convert_payload_0_0_3.py

```diff
@@ -72,3 +72,14 @@ def add_list_collector_answers(
                 answer = answer_store.get_answer(answer_id, list_item_id)
                 if answer:
                     answers_payload.add_or_update(answer)
+
+
+def convert_lists_to_payload_0_0_3(list_store) -> Dict[str, Sequence[str]]:
```

Comment:
This should be returning a list of dicts not a single dict as you can have more the one list in a survey.

Reply:
```Python
def convert_lists_to_payload_0_0_3(list_store) -> Sequence[Mapping]:
    serialised = list_store.serialise()
    lists = [{'name': list_['name'], 'items': list_['items'], 'primary_person': list_.get('primary_person')} for list_ in ]
	
	for list_ in lists:
		if list_['primary_person'] is None:
			del list_['primary_person']
	return lists
```

will do the trick

Reply:
James do you mean:
```Python
def convert_lists_to_payload_0_0_3(list_store) -> Sequence[Mapping]:
    serialised = list_store.serialise()
    lists = [{'name': list_['name'], 'items': list_['items'], 'primary_person': list_.get('primary_person')} for list_ in serialised]
	
	for list_ in lists:
		if list_['primary_person'] is None:
			del list_['primary_person']
	return lists
```
or should I use list_store?

Reply:
Yes, yes I do